editor/sidebar/block-inspector/style.scss

```diff
@@ -34,6 +34,10 @@
 	}
 }
 
+.editor-block-inspector__content .editor-advanced-controls {
+	margin-bottom: -14px;
```

Comment:
Why not `-16px` like in other places? There is still a small whitespace added with the current value:

![screen shot 2017-10-03 at 11 58 06](https://user-images.githubusercontent.com/699132/31119739-349b289e-a832-11e7-8e21-7cb58566d5bc.png)

Reply:
Because of the focus style when clicked.

Reply:
@jasmussen should have a better idea what's best. I'm only asking to learn myself :)